### PR TITLE
Polyhedron demo: Fix polyhedron orientation errors

### LINKED ISSUE #2423: Two-side drawing of a tetrahedron
The color used for one of the face (seen) from the outside is actually the color when seen from the inside. The problem goes away if I add a center vertex in that face (adding a center vertex in another face does not solve the pb). The tetrahedron I used is the default one generated by basic generators.

The bug is observed in master and we should check whether it is also in 4.11

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

Fix orientation of facets in modern flat

## Release Management

* Issue(s) solved (if any): fix #2423

### CODE REVIEW

**Comment by maxGimeno** [comment]
@lrineau the bug is not in 4.11, it is related to the modern flat, which was not in 4.11

File: Polyhedron/demo/Polyhedron/resources/shader_flat.g [hunk 1]
```diff
@@ -18,9 +17,13 @@ out GS_OUT
 } gs_out;
 
 uniform mat4 mv_matrix;
+uniform mat4 mvp_matrix;
 
 void main(void)
 {
+  vec4 norm1 = inverse(mvp_matrix)*gl_in[1].gl_Position - inverse(mvp_matrix)*gl_in[0].gl_Position;
+  vec4 norm2 = inverse(mvp_matrix)*gl_in[2].gl_Position - inverse(mvp_matrix)*gl_in[1].gl_Position;
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/resources/shader_flat.g]: You should store the result of the expression `inverse(mvp_matrix)`.

Comment (lrineau) [reply Polyhedron/demo/Polyhedron/resources/shader_flat.g]: ping @maxGimeno

**Comment by maxGimeno** [comment]
done

**Pull request merged**